app/models/employer.rb

```diff
@@ -0,0 +1,15 @@
+class Employer < User
+  has_many :workers
+
+  def hire_worker(worker_id)
+    worker = Worker.find worker_id
+    worker.employer_id = id
```

Comment:
Most devs shy away from using `id` assignments cause it's an implementation detail. What if all a sudden the worker needs to move out of this app and into another. That would require using `guid` rather than the internal app `id`.

What do you think about assigning the employee itself?
```ruby
worker.employer = self
```